debezium-core/src/main/java/io/debezium/transforms/outbox/SchemaBuilderUtil.java

```diff
@@ -148,4 +148,8 @@ private static Schema buildDocumentUnionSchema(ArrayNode array) {
         }
         return builder.build();
     }
+
+    private static boolean isNotInSchema(SchemaBuilder builder, String fieldName) {
```

Comment:
not sure if it's the best name for this method - I'm happy to change it

Reply:
The method name is fine; although a shorter variant could have been `hasField`.  There is no need to change it really unless Gunnar or Jiri feel strongly otherwise.

Reply:
I went with `hasNotField` - imo longer name doesn't make it less readable but using shorter name and then negating the result returned would make code less readable